google/auth/transport/aiohttp_requests.py

```diff
@@ -31,6 +32,51 @@
 _DEFAULT_TIMEOUT = 180  # in seconds
 
 
+class _CombinedResponse(transport.Response):
+    '''
+    In order to more closely resemble the `requests` interface, where a raw
+    and deflated content could be accessed at once, this class lazily reads the 
+    stream in `transport.Response` so both return forms can be used
+    '''
```

Comment:
Use double quotes for docstring. Does this need to be fleshed out more?

Reply:
Resolved, changed to double quotes with blacken.